### ISSUE TITLE
Implement conditional field display

### ISSUE THREAD

**Issue by bengolder** [codeforamerica/rva-screening#110]
Many of the fields in patient details should not be displayed unless they are needed as follow ups to a previous answer.

These are documented here:
http://codeforamerica.github.io/rva-screening-question-visualizations/

**Comment by bengolder** [comment]
Took a stab at this. It would require adding the slugs for particular fields as a class on the container of the field. It allows us to specify question dependencies as a JSON:

``` javascript
var dependencies = [
  { target: "marital_status", child:"spouse_employment_status", 
    type: "equals", comparator: "MAR" },
  { target: "employment_status", child:"employers",
    type: "in", comparator: [ "FT", "PT", "SEA" ] },
];
```

![conditional](https://cloud.githubusercontent.com/assets/451510/7759520/eda74514-ffc6-11e4-8797-13a64cfa33c4.gif)

**Comment by bengolder** [comment]
I'm happy to make some comments on lines of code in the commit or add more documentation to explain the approach. Let me know if you'd like further explanation.

**Comment by bengolder** [comment]
Closed with #150
